lib/features/app/di/app_scope.dart

```diff
@@ -22,6 +30,12 @@ class AppScope implements IAppScope {
   @override
   late VoidCallback applicationRebuilder;
 
+  @override
+  final FlutterSecureStorage secureStorage = const FlutterSecureStorage();
+
+  @override
+  late final IAuthTokensStorage authTokensStorage = _createAuthTokensStorage();
```

Comment:
can we simply set `= AuthTokensStorageSecure(secureStorage)` without using method?